packages/strapi-parts/src/Combobox/components.js

```diff
@@ -18,13 +19,27 @@ export const MainRow = styled(Row)`
   `
       : undefined}
 
-  &:focus-within {
-    outline: none;
-    box-shadow: 0 0 0 2px rgba(66, 153, 225, 0.6);
-  }
+  ${inputFocusStyle()}
+`;
+
+export const ValueContainer = styled.div`
+  padding: 1px 2px;
```

Comment:
I've added the same input[text] padding to the value container to avoid this : 
![padddding](https://user-images.githubusercontent.com/7756284/137692633-2c7b66ee-f791-41cc-a20f-434a3c619938.gif)